Lib/uuid.py

```diff
@@ -1,8 +1,8 @@
 r"""UUID objects (universally unique identifiers) according to RFC 4122/9562.
 
 This module provides immutable UUID objects (class UUID) and the functions
-uuid1(), uuid3(), uuid4(), uuid5(), uuid6(), and uuid8() for generating
-version 1, 3, 4, 5, 6, and 8 UUIDs as specified in RFC 4122/9562.
+uuid{N}() for generating UUIDs version N as specified in RFC 4122/9562 for
+N = 1, 3, 4, 5, 6, 7, and 8.
```

Comment:
```suggestion
uuid<N>() for generating UUIDs for version N as specified in RFC 4122/9562.
```

Reply:
I would use wording similar to the reST docs, not invent an `<N>` notation that is not seen elsewhere.
You never know who is learning about the module via pydoc, tooltips or reading the source.